api/integrations/launch_darkly/services.py

```diff
@@ -0,0 +1,297 @@
+from contextlib import contextmanager
+from typing import TYPE_CHECKING
+
+from django.conf import settings
+from django.core import signing
+from django.utils import timezone
+from requests.exceptions import HTTPError, RequestException
+
+from environments.models import Environment
+from features.feature_types import MULTIVARIATE, STANDARD
+from features.models import Feature, FeatureState, FeatureStateValue
+from features.multivariate.models import (
+    MultivariateFeatureOption,
+    MultivariateFeatureStateValue,
+)
+from features.value_types import STRING
+from integrations.launch_darkly import types as ld_types
+from integrations.launch_darkly.client import LaunchDarklyClient
+from integrations.launch_darkly.constants import (
+    LAUNCH_DARKLY_IMPORTED_TAG_COLOR,
+)
+from integrations.launch_darkly.models import (
+    LaunchDarklyImportRequest,
+    LaunchDarklyImportStatus,
+)
+from projects.tags.models import Tag
+
+if TYPE_CHECKING:  # pragma: no cover
+    from projects.models import Project
+    from users.models import FFAdminUser
+
+
+def _sign_ld_value(value: str, user_id: object) -> str:
+    return signing.dumps(value, salt=f"ld_import_{user_id}")
+
+
+def _unsign_ld_value(value: str, user_id: object) -> str:
+    return signing.loads(
+        value,
+        salt=f"ld_import_{user_id}",
+        max_age=settings.LAUNCH_DARKLY_IMPORT_SENSITIVE_DATA_MAX_AGE_SECONDS,
```

Comment:
This will fail if the task processor can't load this in less than `LAUNCH_DARKLY_IMPORT_SENSITIVE_DATA_MAX_AGE_SECONDS`(i.e: 30 ) seconds, right? I am not sure if this is a good idea?

Reply:
What would be a sensible max age from your standpoint? Should we have a max age at all?

Reply:
I'd probably just get rid of the max age

Reply:
Done.